src/storage/upgrade_project.ts

```diff
@@ -101,9 +101,9 @@ async function upgradeFrom_001_to_002(
 }
 
 async function upgradeFrom_002_to_003(
-  storage: commonStorage.Storage,
-  projectName: string,
-  projectInfo: storageProject.ProjectInfo): Promise<void> {
+   storage: commonStorage.Storage,
```

Comment:
You need one more space on these 3 lines.

Reply:
done.